# Logical camera image message
Model[] models                  # models detected (poses in the frame of the camera)
geometry_msgs/Pose pose         # camera pose
